index.html

```diff
@@ -0,0 +1,256 @@
+          </div>
+          <div>
+            Manubot is a workflow and set of tools for the next generation of scholarly publishing.
+            Write your manuscript in <a href="https://www.markdownguide.org/">markdown</a>, track it with your favorite <a href="https://git-scm.com/">version control</a>, automatically convert it to <code>.html</code>, <code>.pdf</code>,  or <code>.docx</code>, and deploy it to your <a href="https://pages.github.com/">destination</a> of choice.
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">How It Works</div>
+      <div class="row flex_wrap workflow_row">
+        <div class="workflow_item">
+          <div class="workflow_image"></div>
+          <div>input <code>markdown</code></div>
+        </div>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <div class="workflow_item">
+          <div class="workflow_image"></div>
+          <div>
+            process w/ Manubot tools and <a href="https://pandoc.org/">Pandoc</a>
+          </div>
+        </div>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <div class="workflow_item">
+          <div class="workflow_image"></div>
+          <div>
+            output <code>.html</code>, <code>.pdf</code>, <code>.docx</code>
+          </div>
+        </div>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <div class="workflow_item">
+          <div class="workflow_image"></div>
+          <div>test, deploy to host</div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">The Best Parts</div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-atlas fa-lg"></i> Open, collaborative, versioned
+          </div>
+          <div>
+            Author your manuscript the open-source way.
+            Let anyone propose changes.
+            Iteratively review, discuss, and refine with collaborators.
+            Coordinate teams of any size with ease.
+            Keep track of versions, to-dos, revisions, and who wrote what and when.
+            All in a <a href="https://github.com/">central location</a>.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-magic fa-lg"></i> Automated and continuous
+          </div>
+          <div>
+            Never use a reference manager again.
+            Provide a DOI, ISBN, PubMed ID, url, etc., and let Manubot generate and number citations for you.
```

Comment:
URL ALL CAPS probably.

Possibly add:

Choose from [thousands](https://www.zotero.org/styles "Citation Style Language (CSL) Styles") of existing bibliographic styles.